src/phases/query/CollScanPredicateSelectivity.yml

```diff
@@ -0,0 +1,427 @@
+            onePercentNegativeInt: {^Choose: {from: *PositiveAndNegativeIntegers, weights: [99, 1]}},
+            nested: [
+              {
+                onePercentConstantOne: {^Choose: {from: ["not int", 1], weights: [99, 1]}},
+                tenPercentConstantOne: {^Choose: {from: ["not int", 1], weights: [90, 10]}},
+                quarterConstantOne: {^Choose: {from: ["not int", 1], weights: [75, 25]}},
+                onePercentPositiveInt: {^Choose: {from: *PositiveAndNegativeIntegers, weights: [1, 99]}},
+                tenPercentPositiveInt: {^Choose: {from: *PositiveAndNegativeIntegers, weights: [10, 90]}},
+                quarterPositiveInt: {^Choose: {from: *PositiveAndNegativeIntegers, weights: [25, 75]}},
+                halfPositiveInt: {^Choose: {from: *PositiveAndNegativeIntegers, weights: [50, 50]}},
+                quarterNegativeInt: {^Choose: {from: *PositiveAndNegativeIntegers, weights: [75, 25]}},
+                tenPercentNegativeInt: {^Choose: {from: *PositiveAndNegativeIntegers, weights: [90, 10]}},
+                onePercentNegativeInt: {^Choose: {from: *PositiveAndNegativeIntegers, weights: [99, 1]}},
+              }
+            ]
+          }
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+
+# In actors' names, a 'good case' means that the ordering of predicates defined by actual
+# selectivities matches the ordering defined by heuristic selectivity estimation. Heuristic CE 
+# defines the following selectivity order for predicates:
+# $eq < closed range (e.g. {$gt: 10, $lte: 20}) < open range (e.g. {$gt: 10}) < $ne
+
+- ActorFromTemplate:
+    TemplateName: FindQueryTemplate
+    TemplateParameters:
+      Name: HighSelectivityGoodCaseTwoClauses
+      ActivePhase: 3
+      Filter:
+        # Here (0, 1e9) is the most selective predicate, both in reality and according to heuristic CE.
+        # Note that without predicate reordering, predicates are ordered by first comparing field names.
+        &HighSelectivityGoodCaseTwoClauses {
+          onePercentPositiveInt: {$gt: 0, $lt: *Billion}, # Selectivity = .01
+          onePercentNegativeInt: {$gt: 0}, # Selectivity = .99
+        }
+
+- ActorFromTemplate:
+    TemplateName: FindQueryTemplate
+    TemplateParameters:
+      Name: MediumSelectivityGoodCaseTwoClauses
+      ActivePhase: 4
+      Filter:
+        &MediumSelectivityGoodCaseTwoClauses {
+          tenPercentPositiveInt: {$gt: 0, $lt: *Billion}, # Selectivity = .1
+          tenPercentNegativeInt: {$gt: 0}, # Selectivity = .9
+        }
+
+- ActorFromTemplate:
+    TemplateName: FindQueryTemplate
+    TemplateParameters:
+      Name: LowSelectivityGoodCaseTwoClauses
+      ActivePhase: 5
+      Filter:
+        &LowSelectivityGoodCaseTwoClauses {
```

Comment:
I hate making the names even longer, but... could we do something like:
- GoodCase -> HeuristicGood
- Selectivity -> Sel

I think this would help when reading dashboard metrics